pennylane/drawer/tape_text.py

```diff
@@ -64,190 +60,148 @@ class _Config:
-        min_w, max_w = min(mapped_wires), max(mapped_wires)
-        layer_str[min_w] = "╭"
-        layer_str[max_w] = "╰"
 
-        for w in range(min_w + 1, max_w):
-            layer_str[w] = "├" if w in mapped_wires else "│"
-
-    return layer_str
-
-
-def _add_cond_grouping_symbols(op, layer_str, config):
-    """Adds symbols indicating the extent of a given object for conditional
-    operators"""
-    n_wires = len(config.wire_map)
-
-    mapped_wires = [config.wire_map[w] for w in op.wires]
-    mapped_bits = [config.bit_map[m] for m in op.meas_val.measurements]
-    max_w = max(mapped_wires)
-    max_b = max(mapped_bits) + n_wires
+def _initialize_wire_and_bit_totals(
+    config: _Config, show_wire_labels: bool
+) -> tuple[list[str], list[str]]:
+    """Initialize the wire totals and bit_totals with the required wire labels."""
+    if show_wire_labels:
+        wire_totals = [f"{wire}: " for wire in config.wire_map]
+    else:
+        wire_totals = [""] * len(config.wire_map)
+    line_length = max(len(s) for s in wire_totals)
 
-    ctrl_symbol = "╩" if config.cur_layer != config.cwire_layers[max(mapped_bits)][-1] else "╝"
-    layer_str[max_b] = f"═{ctrl_symbol}"
+    wire_totals = [s.rjust(line_length, " ") for s in wire_totals]
+    bit_totals = [" " * line_length for _ in config.bit_map]
+    return wire_totals, bit_totals
 
-    for w in range(max_w + 1, max(config.wire_map.values()) + 1):
-        layer_str[w] = "─║"
 
-    for b in range(n_wires, max_b):
-        if b - n_wires in mapped_bits:
-            intersection = "╣" if config.cur_layer == config.cwire_layers[b - n_wires][-1] else "╬"
-            layer_str[b] = f"═{intersection}"
-        else:
-            filler = " " if layer_str[b][-1] == " " else "═"
-            layer_str[b] = f"{filler}║"
+def _initialize_layer_str(config: _Config) -> list[str]:
+    """Initialize the list of strings for a new layer.
 
-    return layer_str
+    For example, we have three wires and two classical wires, we will get:
 
+    .. code-block::
 
-def _add_mid_measure_grouping_symbols(op, layer_str, config):
-    """Adds symbols indicating the extent of a given object for mid-measure
-    operators"""
-    if op not in config.bit_map:
-        return layer_str
+        ['─', '─', '─', ' ', ' ']
 
+    """
     n_wires = len(config.wire_map)
-    mapped_wire = config.wire_map[op.wires[0]]
-    bit = config.bit_map[op] + n_wires
-    layer_str[bit] += " ╚"
-
-    for w in range(mapped_wire + 1, n_wires):
-        layer_str[w] += "─║"
+    n_bits = len(config.bit_map)
 
-    for b in range(n_wires, bit):
-        filler = " " if layer_str[b][-1] == " " else "═"
-        layer_str[b] += f"{filler}║"
+    # Create initial strings for the current layer using wire and cwire fillers
+    layer_str = [config.wire_filler] * n_wires + [" "] * n_bits
+    for b in config.bit_map.values():
+        layer_str[b + n_wires] = config.bit_filler(b)
```

Comment:
Super minor, but couldn't we just skip adding those `" "` entries for the classical bits only to overwrite them immediately after? Is this to have the same list size from the beginning on? :thinking: 
I.e., couldn't we do
```suggestion
    layer_str = [config.wire_filler] * n_wires + [config.bit_filler(b) for b in config.bit_map.values()]
```
or a variant thereof that computes the second sublist in its own line?